mobile.css

```diff
@@ -0,0 +1,128 @@
+.display {
+  display: block;
+}
+
+.menu-icon {
+  background: #6070ff;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  padding: 50px 30px;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(12px);
```

Comment:
- [ ] This part is really tricky if you are wondering why backdrop-filter not working with you. I went ahead and investigated that for you 🤓. It's because we have our `div` for menu inside the header if you want it to blur the entire page like in Figma design let's make sure we have this div direct child to `body` tag, Kindly make this to match the Figma design.
note: there are some issues you will face when you implement that, I don't want to spoil the fun of fixing it 😇, Just letting you know

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/157855976-813a60e1-8d23-44b8-8126-f8f942474d93.png) | ![image](https://user-images.githubusercontent.com/87689547/157856061-37070675-53d6-4350-b079-789400466bfa.png)

Reply:
* Issue resolved.